lab-glen/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "lab-glen",
+  "version": "1.0.0",
+  "description": "![cf](https://i.imgur.com/7v5ASc8.png) lab 16 express basic authorization ======",
+  "main": "server.js",
+  "scripts": {
```

Comment:
"test": "./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha",
    "coveralls": "cat ./coverage/lcov.info | ./node_modules/.bin/coveralls ",
    "debug": "DEBUG=turnipVentures* npm run test",
    "lint": "./node_modules/eslint/bin/eslint.js .",
    "start": "node server.js"


I had some issues with your scripting, again there is some differences but it could be that I was on branch 19 and there were changes that kept kicking me